library/Vanilla/Theme/JsonAsset.php

```diff
@@ -41,9 +41,18 @@ public function asArray(): array {
     /**
      * Get the JSON content.
      *
-     * @return string
+     * @param string $type
+     * @return mixed
      */
-    public function getData(): string {
-        return $this->data;
+    public function getData(string $type = 'string') {
+        if ($type === 'array') {
+            return json_decode(json_encode($this->data), true);
```

Comment:
I am not sure why we need to `encode` and `decode`, could we at least try to return it as is if that is an array already?
```suggestion
            if (is_array($this->data)) {
                return $this->data;
            } else {
                return json_decode(json_encode($this->data), true);
            }
```

Reply:
Another note, any time we are encoding/decoding we should be passing `JSON_UNESCAPED_UNICODE` so that any unicode characters don't get messed up.